ci/docker/base.test.wanda.yaml

```diff
@@ -6,5 +6,6 @@ srcs:
   - .bazeliskrc
 build_args:
   - REMOTE_CACHE_URL
+  - BUILDKITE_BAZEL_CACHE_URL
```

Comment:
is the value of this default inside wanda?

Reply:
no, but it is set by the `rayci` command, the value is from the rayci pipeline config.